Makefile

```diff
@@ -1,4 +1,4 @@
-ORG     := $(shell basename $(realpath ..))
+ORG     ?= $(shell basename $(realpath ..))
```

Comment:
Set `ORG` env var in your CircleCI build environment so that you can actually test this workflow on your GitHub/Quay/CircleCI account.